back/app/services/custom_field_params_service.rb

```diff
@@ -9,7 +9,7 @@ def custom_field_values_params(allowed_fields)
         fields_with_array_keys[field.key.to_sym] = []
       when 'file_upload'
         fields_with_array_keys[field.key.to_sym] = %i[id content name]
-      when 'point'
+      when 'point', 'line', 'polygon'
         fields_with_array_keys[field.key.to_sym] = [:type, { coordinates: [] }]
```

Comment:
Not 100% sure what this service does. I'm assuming we actually might need a separate line here for line and polygon though?

Reply:
I can't remember what the service does either. I'll investigate.

Reply:
It's a matcher that permits certain formats of params, and it's where the problem lies in params with nested arrays (which, for some reason, Rails strong params don't allow)